test/suite/test_oligarch04.py

```diff
@@ -70,7 +61,7 @@ def test_oligarch04(self):
             cursor["Hi " + str(i)] = "There"
             cursor["OK " + str(i)] = "Go"
             if i % 10000 == 0:
-                time.sleep(1)
+                time.sleep(5)
```

Comment:
Why is this necessary?

Reply:
That's a very good question - I do know that if the delay is not there, the count at the end is incorrect (I guess some items aren't available).  hmmm - these delays have been in the tests for a while.  I'll ping our slack channel.

Reply:
Can you please create a ticket that describes this failure?